boards/Makefile.common

```diff
@@ -281,6 +281,38 @@ lst: $(TARGET_PATH)/release/$(PLATFORM).lst
 show-target:
 	$(info $(TARGET))
 
+# Additional flags that can be passed to cargo bloat via an
+# environment variable. Allows users to pass arbitrary flags
+# supported by cargo bloat to customize the output. By default,
+# pass an empty string.
+BLOAT_FLAGS ?=
+
+
+# cargo bloat does not support -Z build-std, so we must remove it from cargo flags
+# see https://github.com/RazrFalcon/cargo-bloat/issues/62
+CARGO_FLAGS_TOCK_NO_BUILD_STD := $(filter-out -Z build-std=core,$(CARGO_FLAGS_TOCK))
```

Comment:
These should be moved up in the file, and should be included in our debug output.